### PR TITLE
Small changes to custominvoice texts suggested on tampere channels

### PR DESCRIPTION
**Pull request by tswfi**
Some changes were suggested to the custom invoice texts

https://matrix.to/#/!RsUTIzdZIWSNaRSEIe:hacklab.fi/$JsA4SHorwwmjhX-fVUPcVu9wjfH11HHZiiMawKsTv3c?via=hacklab.fi&via=irc.snt.utwente.nl&via=matrix.org

Also tried to make it clearer that one should not use the custom invoices all the time

### CODE REVIEW

File: www/templates/www/custominvoice.html [hunk 1]
```diff
@@ -36,7 +40,7 @@ <h2>{% trans 'Pending invoices'%}</h2>
     <td>{{ custominvoice.days }} {% trans 'days of' %} {{ custominvoice.subscription.service.name }}</td>
     <td>{{ custominvoice.amount }}</td>
     <td>{{ custominvoice.reference_number }}</td>
-    <td><a href="{% url 'custominvoice_action' action='delete' invoiceid=custominvoice.id %}">{% trans 'delete' %}</a></td>
+    <td><a onclick="return confirm('{% blocktrans %}Are you sure? Please note, if you have already paid the invoice but the payment is not registered yet deleting the custom invoice will never be targeted{% endblocktrans %}')" href="{% url 'custominvoice_action' action='delete' invoiceid=custominvoice.id %}">{% trans 'delete' %}</a></td>
```
Comment (brndd) [inline www/templates/www/custominvoice.html:43]: I think this sentence could use a comma ("...but the payment is not registered yet, deleting the..."). I also don't understand what "targeted" means here.

Comment (tswfi) [reply www/templates/www/custominvoice.html:43]: "kohdistua", couldn't think of a better word...

We've had at least few persons who created custom invoice on day 1, paid on day2 and deleted on day3 and then bank transactions were read in on day 7 or something like that but of course that wont mark anything for the user and is just considered a donation at that point...

Comment (brndd) [reply www/templates/www/custominvoice.html]: Oh, I see. Something like "Please note: deleting a custom invoice before the payment for it is processed by the system will cause the payment to fail", then?

Comment (tswfi) [reply www/templates/www/custominvoice.html]: well the payment will still come through but it wont map to the user and will just be a donation :D 
anyways I removed the whole inline js part as Olmari reminded me why we don't do that :D

**Comment by tswfi** [comment]
rebased and force pushed to get rid of pyup warnings

**Comment by tswfi** [comment]
Next PR can make these texts even better :)

**Pull request merged**